pkg/scheduler/api/types.go

```diff
@@ -26,10 +26,6 @@ import (
 )
 
 const (
-	// MaxUint defines the max unsigned int value.
-	MaxUint = ^uint(0)
-	// MaxInt defines the max signed int value.
-	MaxInt = int(MaxUint >> 1)
 	// MaxTotalPriority defines the max total priority value.
```

Comment:
Should other consts be moved outside `types.go` as well? Where should they belong then?

Reply:
We can leave them here as this package isn't supposed to be exposed as external API. Instead, 
 the package under `staging` will be exposed.